src/coreclr/nativeaot/Runtime.Base/src/System/Runtime/ExceptionHandling.cs

```diff
@@ -550,6 +550,7 @@ internal object ThrownException
 #if NATIVEAOT
         [RuntimeExport("RhThrowHwEx")]
 #endif
+        [StackTraceHidden]
```

Comment:
this is only updating nativeaot, is the same file reused for coreclr as well?

Reply:
This is the common EH code so here it updates both the coreclr and nativeaot behavior.